docs/source/markdown/podman.1.md

```diff
@@ -75,110 +72,159 @@ If no identity file is provided and no user is given, podman defaults to the use
+
+Storage driver. 
+
+**Note:**
+- Default is *vfs* for non-root users when *fuse-overlayfs* is not available.  
 
-Overriding this option will cause the *storage-opt* settings in /etc/containers/storage.conf to be ignored.  The user must
+- The `STORAGE_DRIVER` environment variable overrides the default.  The `--storage-driver` specified driver overrides all.
+
+- Overriding this option will cause the *storage-opt* settings in `/etc/containers/storage.conf` to be ignored.  The user must
 specify additional options via the `--storage-opt` flag.
 
 #### **--storage-opt**=*value*
 
-Storage driver option, Default storage driver options are configured in /etc/containers/storage.conf (`$HOME/.config/containers/storage.conf` in rootless mode). The `STORAGE_OPTS` environment variable overrides the default. The --storage-opt specified options overrides all. If you specify --storage-opt="", no storage options will be used.
+| User      | Value (Default)|
+| ----------- | ----------- |
+| Root        | `/etc/containers/storage.conf`|
+| Rootless    | `$HOME/.config/containers/storage.conf`|
+
+Storage driver option.
+
+The `STORAGE_OPTS` environment variable overrides the default. The --storage-opt specified options overrides all. If you specify `--storage-opt=""`, no storage options will be used.
 
-#### **--syslog**=*true|false*
+#### **--syslog**=*true| **false***
 
-Output logging information to syslog as well as the console (default *false*).
+| Value      | Description |
+| ----------- | ----------- |
+| True      | Information is logged |
+| False   | Information is not logged |
 
-On remote clients, logging is directed to the file $HOME/.config/containers/podman.log.
+Output logging information to syslog as well as the console.
+
+On remote clients, logging is directed to the file `$HOME/.config/containers/podman.log`.
 
 #### **--tmpdir**
 
 Path to the tmp directory, for libpod runtime content.
 
-NOTE --tmpdir is not used for the temporary storage of downloaded images.  Use the environment variable `TMPDIR` to change the temporary storage location of downloaded container images. Podman defaults to use `/var/tmp`.
+**Note:** 
+- --tmpdir is not used for the temporary storage of downloaded images.  Use the environment variable `TMPDIR` to change the temporary storage location of downloaded container images. Podman defaults to use `/var/tmp`.
 
 #### **--version**, **-v**
 
-Print the version
+Print the version.
 
 ## Environment Variables
 
-Podman can set up environment variables from env of [engine] table in containers.conf. These variables can be overridden by passing  environment variables before the `podman` commands.
+Podman can set up environment variables from env of [engine] table in `containers.conf`. These variables can be overridden by passing  environment variables before the `podman` commands.
 
 ## Remote Access
 
 The Podman command can be used with remote services using the `--remote` flag. Connections can
```

Comment:
The format of the OPTIONS should stay the same throughout the whole document. In this case **--remote** should be **bold**. 
The term `flag` should be replaced with the term OPTION.